install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml

```diff
@@ -296,7 +296,7 @@ spec:
           - serviceAccountToken:
               path: istio-token
               expirationSeconds: 43200
-              audience: {{ $.Values.global.trustDomain }}
+              audience: {{ $.Values.global.sds.token.aud }}
```

Comment:
this is a backwards incompatible change it seems?

Reply:
hmm there should be any problem, but regardless of this, sds is an alpha feature and not many people are using it seriously

Reply:
And it's clear that trustDomain is not the real audience of this token, but the CA service that provides the certificates.